akvo/rsr/models/transaction.py

```diff
@@ -162,8 +162,17 @@ def receiver_organisation_show_link(self):
                                                    self.receiver_organisation.name)
         return ''
 
+    def get_currency(self):
```

Comment:
Could be rewritten to:
`return self.currency if self.currency else self.project.currency`